src/managing-workflow/upgrading-workflow.md

```diff
@@ -89,6 +89,14 @@ $ kubectl --namespace=deis get secret database-creds -o yaml > ~/active-deis-dat
 $ kubectl --namespace=deis get secret builder-ssh-private-keys -o yaml > ~/active-deis-builder-secret-ssh-private-keys.yaml
 ```
 
+#### Optional: Fetch platform SSL credentials
+Installations that have installed a [platform SSL](https://deis.com/docs/workflow/managing-workflow/platform-ssl/) certificate and private key on the router will need to save those secrets locally as well. They will be installed at a later step with the other credentials saved to the local workstation.
+
+``` 
+# fetch the platform SSL certificate and private key and store on the local workstation
+$ kubectl --namespace=deis get secret deis-router-platform-cert -o yaml > ~/active-deis-router-platform-cert
```

Comment:
Fixed, thanks for catching. Also amended the commit message to use the correct (I hope) format.